src/GafferUI/ContextTracker.cpp

```diff
@@ -316,7 +448,8 @@ void ContextTracker::update()
 			continue;
 		}
 
-		Context::Scope scopedContext( context.get() );
+		Context::EditableScope scopedContext( context.get() );
+		scopedContext.setCanceller( canceller );
```

Comment:
It took me a bit of staring to figure out why it was necessary to do this ( and then copy contexts with `omitCanceller = true` ) - but I think I get it now: we don't want to store this canceller, but we do want the getValue() calls for enabledPlug and selectorPlug to have the proper canceller set. Might be worth a comment?